src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```
@@ -304,3 +304,33 @@ details.sd-dropdown {
     }
   }
 }
+
+/*******************************************************************************
+* Buttons (which in Sphinx Design are actually links that look like buttons)
```

```suggestion
* Buttons (which in Sphinx Design are actually links that look like buttons)
* ref: https://sphinx-design.readthedocs.io/en/pydata-theme/badges_buttons.html#buttons
```
just linking to the docs for reference